packages/gatsby-recipes/src/index.js

```diff
@@ -1,7 +1,7 @@
-exports.startGraphQLServer = require(`./graphql-server`).startGraphQLServer
-
 // data = { recipe?: string, graphqlPort: number, projectRoot: string }
 module.exports = async data => {
   const cli = require(`import-jsx`)(require.resolve(`./cli`))
   return cli(data)
 }
+
+module.exports.startGraphQLServer = require(`./graphql-server`).startGraphQLServer
```

Comment:
Is this legal commonjs? You're effectively making startGraphQLServer a static property of the cli function

Reply:
Are we able to refactor this to make the cli a default export (and maybe even make it an es module or TypeScript)? Where is it imported?